templates/master/00-master/baremetal/files/baremetal-keepalived-keepalived.yaml

```diff
@@ -3,9 +3,19 @@ path: "/etc/kubernetes/static-pod-resources/keepalived/keepalived.conf.tmpl"
 contents:
   inline: |
     vrrp_script chk_ocp {
-        script "/usr/bin/curl -o /dev/null -kLfs https://localhost:6443/readyz && /usr/bin/curl -o /dev/null -kLfs http://localhost:50936/readyz"
+        script "/usr/bin/curl -o /dev/null -kLfs https://localhost:9443/readyz"
```

Comment:
So we change to only chk_ocp to only check the API status through the load balancer.